src/go/vtunnel/cmd/peer_linux.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/rancher-sandbox/rancher-desktop/src/go/vtunnel/pkg/vmsock"
+	"github.com/spf13/cobra"
+)
+
+const (
+	localhost       = "127.0.0.1"
+	defaultPeerPort = 9779
```

Comment:
Rather than having a default, we should just abort if it's not specified: it's always an error if two users of this infrastructure share a port.

Contrast this to the IP address, where it is basically always `127.0.0.1` and being configurable isn't really that useful.

Reply:
True, if the port is already occupied user will get an error which then you will have to provide another port at least you will have that option.
Having these parameters configurable is beneficial, if we are testing or debugging. I would rather pass in arguments through CLI when debugging something rather than editing the code/building/deploying 🤷

Reply:
> Rather than having a default, we should just abort if it's not specified: it's always an error if two users of this infrastructure share a port.

Is there a reason the port needs to be specified at all? Why are you not picking an available port to use, and report that back to the caller?

Reply:
This needs to be synchronized between the peer (Linux side) and host (Windows side); so passing it in as arguments to both would be easier than trying to get the two to sync.